clients/cli/src/main/java/org/apache/gravitino/cli/FullName.java

```diff
@@ -73,10 +73,7 @@ public String getMetalakeName() {
       }
     }
 
-    // Extract the metalake name from the full name option
-    if (line.hasOption(GravitinoOptions.NAME)) {
-      return line.getOptionValue(GravitinoOptions.NAME).split("\\.")[0];
-    }
+    System.err.println(ErrorMessages.MISSING_METALAKE);
```

Comment:
You should also exit here

Reply:
The code that follows handles null，and give the information as follows 
`The argument 'metalakeName' must be a valid name`